karma.conf.js

```diff
@@ -1,10 +1,10 @@
-module.exports = function (config) {
+module.exports = function(config) {
```

Comment:
Let's keep the style consistent and keep the eslint config at the top so all files get linted